core/vm/contracts.go

```diff
@@ -75,6 +76,14 @@ var (
 	hashHeaderAddress            = celoPrecompileAddress(9)
 	getParentSealBitmapAddress   = celoPrecompileAddress(10)
 	getVerifiedSealBitmapAddress = celoPrecompileAddress(11)
+	// BLS12-377
```

Comment:
I am still optimistic that we can include 2539 in Berlin, in which case these would change

Reply:
I am no longer optimistic.